src/app/app.module.ts

```diff
@@ -9,6 +9,7 @@ import { HomeComponent } from './components/home/home.component';
 import { TimesheetModule } from './modules/timesheet/timesheet.module';
 import { ExpenseModule } from './modules/expense/expense.module';
 import { NotificationModule } from './modules/notification/notification.module';
+import {HttpClientModule} from '@angular/common/http';
```

Comment:
Try to keep the same code formatting as the rest of the file :) 
`{ HttpClientModule }` or install vscode instead of intellij :trollface: